tests/drivers/build_all/ethernet fails to build on mimxrt1062_fmurt6

**Describe the bug**

The tests/drivers/build_all/ethernet test fails to build for mimxrt1062_fmurt6 target. 

`nxp_enet_mdio: undefined reference to __device_dts_ord_445
`

 - What target platform are you using? `mimxrt1062_fmurt6`
 - bisect: e78811bcd6d028437112ba24e237e1ac58094345

**To Reproduce**

Steps to reproduce:
`west build -p -b mimxrt1062_fmurt6/mimxrt1062 tests/drivers/build_all/ethernet -T net.ethernet.build.default
`

**Expected behavior**
Test should not fail.

**Impact**
Could block CI if this test is selected

--- a/boards/nxp/mimxrt1062_fmurt6/mimxrt1062_fmurt6.dts
+++ b/boards/nxp/mimxrt1062_fmurt6/mimxrt1062_fmurt6.dts
@@ -385,12 +385,17 @@
 	nxp,prescaler = <64>;
 };
 
+&enet2 {
+	status = "okay";
+};
+
 &enet2_mac {
 	pinctrl-0 = <&pinmux_enet>;
 	pinctrl-names = "default";
 	zephyr,random-mac-address;
 	phy-connection-type = "rmii";
 	phy-handle = <&phy>;
+	status = "okay";
 };
 
 &enet2_mdio {

